miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/errors/MiniAppErrorTypes.kt

```diff
@@ -0,0 +1,13 @@
+package com.rakuten.tech.mobile.miniapp.errors
+
+/** Enumeration that is used to return  Access Token error. **/
+enum class AccessTokenErrorType(val type: String) {
+    // Requested Audience is not supported
+    AudienceNotSupportedError("AudienceNotSupportedError"),
```

Comment:
I'm not sure if these need to be defined as an enum. Shouldn't it be enough to define them as just strings since the `MiniAppError` accepts a string as the `type`?